modules/cluster-logging-log-store-status-viewing.adoc

```diff
@@ -101,31 +101,29 @@ status: <1>
   shardAllocationEnabled: all
```

Comment:
Should line 49 be `[source,yaml]` since the command is outputting YAML (and that way you get syntax highlighting)?

Reply:
Very good call, I will update this 🙂